src/dao/messageDao.js

```diff
@@ -0,0 +1,84 @@
+var MessageDB = require('../model/messageModel');
+var validationMessage = require('../utils/validations');
+var ListMessagesModelDB = require('../model/listMessagesModel');
+var listMessageDao = require('../dao/listMessagesDao');
+class MessageDao {
+
+    constructor() {
+        var validacion = new validationMessage();
+    }
+
+    getMessagesByListMessageId(listMessageId) {
+        const getMessagesByListMessagesIdPromise = new Promise((resolve, reject) => {
+            MessageDB.find({ '_id': listMessageId }, function (err, messages) {
+                if (err) {
+                    reject(err);
+                } else {
+                    resolve(messages);
+                }
+            });
+        });
+        return getMessagesByListMessagesIdPromise;
+    };
+
+    createMessage(datoMessage,receiverId) { 
+        var validaciones = new validationMessage();
+        const createMessagePromise = new Promise((resolve, reject) => {
+            if (validaciones.isEmptyObject(datoMessage.senderId)) {
+                reject({ "status": "Error: senderId is missing" });
+            } else if (validaciones.isEmptyString(datoMessage.senderId)) {
+                reject({ "status": "Error: senderId is empty" });
+            } else if (validaciones.isEmptyObject(datoMessage.message)) {
+                reject({ "status": "Error: message is missing" });
+            } else if (validaciones.isEmptyString(datoMessage.message)) {
+                reject({ "status": "Error: message is empty" });
+            } else if (validaciones.isEmptyObject(receiverId)) {
+                reject({ "status": "Error: receiverId is missing" });
```

Comment:
What is the difference in missing and empty, for a common user it will be the same?